docs/tutorials/vagrant.rst

```diff
@@ -0,0 +1,740 @@
+
+The `vQuanta_install_centos.json` sample
+
+.. code-block:: JSON
+
+    {
+        "friendlyName": "vQuanta D51 Default Install CentOS",
+        "injectableName": "Graph.DefaultVQuanta.InstallCentOS",
+        "options": {
+          "defaults": {
+            "version": "7",
+            "dnsServers": ["8.8.8.8", "8.8.4.4"],
+            "repo": "{{api.server}}/Centos/7.0",
+            "rootPassword": "rootpassword",
+            "users": [{
+              "name": "rackhd",
+              "password": "rackhd123",
+              "uid": 1010
+            }]
+            },
+          "install-centos": {
+            "schedulerOverrides": {
+              "timeout": 3600000
+            }
+          }
+        },
+        "tasks": [
+            {
+                "label": "create-ipmi-obm-settings",
+                "taskDefinition": {
+                    "friendlyName": "Create IPMI OBM settings",
+                    "injectableName": "Task.Obm.Ipmi.CreateSettings",
+                    "implementsTask": "Task.Base.Obm.Ipmi.CreateSettings",
+                    "options": {
+                        "ipmichannel": null,
+                        "user": "admin",
+                        "password": "admin"
+                    },
+                    "properties": {
+                    }
+                }
+            },
+            {
+                "label": "install-coreos",
+                "taskName": "Task.Os.Install.CentOS",
+                "waitOn": {
+                    "create-ipmi-obm-settings": "succeeded"
+                }
+            }
+        ]
+    }
+
+- add the SKU definition for our simulated hardware::
+
+    cd ~/src/rackhd/example
+    # make sure you're in the example directory to reference the sample JSON correctly
+
+    curl -H "Content-Type: application/json" \
+    -X POST --data @samples/vQuanta_d51_sku.json \
+    http://localhost:9090/api/1.1/skus
+
+The `vQuanta_d51_sku.json` sample
+
+.. code-block:: JSON
+
+    {
+        "name": "vQuanta D51 SKU",
+        "discoveryGraphName": "Graph.DefaultVQuanta.InstallCentOS",
+        "discoveryGraphOptions": {},
+        "rules": [
+            {
+                "path": "dmi.System Information.Product Name",
+                "equals": "D51B-2U (dual 10G LoM)"
+            },
+            {
+                "path": "dmi.System Information.Serial Number",
+                "equals": "SerialNumber"
+            }
+        ]
+    }
+
+Now if you destroy and recreate the simulated hardware, after it is finished with
+discovery, it will automatically install CentOS.
+
+.. warning:: **SLOW ON VAGRANT**
+
+    The simulated hardware is a virtual machine inside another virtual machine, so
+    while this process works, it is very, very slow on most desktops. To see it operational
+    you will want probably want watch the console on the simulated hardware using VNC.
+
+    On my laptop, this process takes approximately 2 hours to fully complete. Using
```

Comment:
Any thoughts on adding some details around these anecdotal reference points?  Knowing the stats of your laptop might help someone gauge how slow it is going to be for them vs. how much they think you deserve a new laptop to speed things up